x-pack/plugins/siem/public/management/common/routing.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { generatePath } from 'react-router-dom';
+import {
+  MANAGEMENT_ROUTING_ENDPOINTS_PATH,
+  MANAGEMENT_ROUTING_POLICIES_PATH,
+  MANAGEMENT_ROUTING_POLICY_DETAILS_PATH,
+  MANAGEMENT_ROUTING_ROOT_PATH,
+} from './constants';
+import { ManagementSubTab } from '../types';
+import { SiemPageName } from '../../app/types';
+
+export type GetManagementUrlProps = {
+  /**
+   * Exclude the URL prefix (everything to the left of where the router was mounted.
+   * This may be needed when interacting with react-router (ex. to do `history.push()` or
+   * validations against matched path)
+   */
+  excludePrefix?: boolean;
+} & (
+  | { name: 'default' }
```

Comment:
not that you need to, but you could replace 

```
  | { name: 'endpointList' }
  | { name: 'policyList' }
  | { name: 'default' }
```
 with
```
 | { name: 'endpointList' | 'policyList' | 'default' }
```

Reply:
True.  Will do it in my next PR. I wanted to maintain clarity on what the different named URLs were and their supported params and your proposed change here still maintains that (ex. `policyDetails` is still defined separately.)

Will make this change in my next PR, since its minor :)